### ISSUE TITLE
As SM, I want to be able to update tags on Bundles in Atlas

### ISSUE THREAD

**Issue by kreynen** [CuBoulder/express#1066]
## Context

With #898 we've altered the SO's view of bundles, but the only way to tag bundles is with Postman.  Since the decision as to when to move bundles between the tabs and expose them to Express users isn't something development should have to be involved in, we should add a UI to ~~the webcentral bundle~~ Shrugger to handle this.  

## Process

1. Login into /webcentral as ADMIN or DEV
2. Click Web Central Tools
3. Click Manage Addon Bundles
4. Select bundle
5. Update tag from options in select list
6. Save

## Expected result

All sites on that environment (dev/test/prod) should see the change immediately in admin/settings/bundles/list

## Current result

NA

**Comment by kreynen** [comment]
Issue moved to [CuBoulder/lil_shrugger #388](https://github.com/CuBoulder/lil_shrugger/issues/388) via [**ZenHub**](https://www.zenhub.com/)
